src/modality-specific-files/stimuli.md

```diff
@@ -0,0 +1,161 @@
+# Stimuli
+
+## Stimulus Files Organization
+
+Stimulus files MUST be stored in the `/stimuli` directory under the root directory of the dataset.
+The `/stimuli` directory can contain subdirectories to organize the stimulus files.
+Stimulus files MUST follow the BIDS naming conventions and are referenced in the `events.tsv`
+file using the `stim_id` column.
+
+The standardization of stimulus files and their annotations within BIDS offers several key benefits:
+
+1.  **Consistency**: Ensures uniform storage and referencing across datasets
+1.  **Reusability**: Enables stimulus reuse across studies through standardized structure
+1.  **Efficiency**: Minimizes redundancy by centralizing annotations
+1.  **Flexibility**: Facilitates dataset reuse with alternative annotations
+
+To preserve backward compatibility with existing datasets (see the Legacy section below), the use of these specifications for the `/stimuli` directory and the `stim_id` column in the `events.tsv` files is RECOMMENDED but not required. Researchers are encouraged to follow these guidelines to enhance the interoperability and reproducibility of their studies.
+
+Following these guidelines will help ensure that stimulus files and their annotations are stored and referenced consistently across different datasets, facilitating data sharing, reuse, and reproducibility.
+
+## File Organization
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+   "stimuli": {
+    "stimuli.tsv": "",
+    "stimuli.json": "",
+    "[stim-<label>[_part-<label>]_<suffix>.<extension>]": "",
+    "[stim-<label>[_part-<label>]_<suffix>.json]": "",
+    "[[stim-<label>_]annotations.tsv]": "",
+    "[[stim-<label>_]annotations.json]": "",
+    "[stim-<label>[_part-<label>]_annot-<label>_events.tsv]": "",
+    "[stim-<label>[_part-<label>]_annot-<label>_events.json]": ""
+    }
+  }) }}
+
+Note: The presence of the `stimuli.tsv` file indicates that the content of the `/stimuli` directory follows this BIDS specification for stimulus organization. This structure is planned to become mandatory in BIDS 2.0.
```

Comment:
```suggestion
Note: The presence of the `stimuli.tsv` file indicates that the content of the `/stimuli` directory follows this BIDS specification for stimulus organization.
```

I do not think we have yet specific plan for that specific aspect... If there is one, let's point to the issue.